client/aws_credentials_test.go

```diff
@@ -7,9 +7,8 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-const awsCredentialsName = "credential_test"
-
 var _ = Describe("AwsCredentials", func() {
+	const awsCredentialsName = "credential_test"
 	var apiKey ApiKey
```

Comment:
Why is that? This applies to all other unrelated tests

Reply:
those were in the global scope so you could get them from other tests file
if i want to declare `dummyProjectId` i couldn't since its already been declared
so i put them inside the test scope

Reply:
> those were in the global scope so you could get them from other tests file

🤮